### PR TITLE
LPS-137754 Limit size of nodes labels

### PR DESCRIPTION
**Pull request by naraandrade**
Story: [LPS-135890](https://issues.liferay.com/browse/LPS-135890)

### CODE REVIEW

**Review by ramonmsp — APPROVED** [review]

File: modules/apps/portal-workflow/portal-workflow-instance-tracker-web/src/main/resources/META-INF/resources/js/util/util.js [hunk 1]
```diff
@@ -22,6 +22,11 @@ import JoinNode from '../components/nodes/hexagon-nodes/JoinNode';
 import JoinXorNode from '../components/nodes/hexagon-nodes/JoinXorNode';
 import BorderStateNode from '../components/nodes/state-node/BorderStateNode';
 import StateNode from '../components/nodes/state-node/StateNode';
+import Edge from '../components/transitions/Edge';
+
+const edgeTypes = {
+	custom: Edge,
+};
```
Comment (rafaprax) [inline modules/apps/portal-workflow/portal-workflow-instance-tracker-web/src/main/resources/META-INF/resources/js/util/util.js:29]: Hey @naraandrade, What do you think if we rename this to `transition` or something similar? Thx

Comment (rafaprax) [reply modules/apps/portal-workflow/portal-workflow-instance-tracker-web/src/main/resources/META-INF/resources/js/util/util.js:29]: `custom` is too generic

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**